dspace/config/modules/matomo.cfg

```diff
@@ -0,0 +1,39 @@
+#---------------------------------------------------------------#
+#----------------MATOMO TRACKER CONFIGURATION-------------------#
+#---------------------------------------------------------------#
+
+# Enables matomo integration mapped inside the MatomoEventListener
+matomo.enabled = false
+# Configured `siteid` inside the matomo dashboard
+matomo.request.siteid = 1
+# Configured tracker.url that will be used by the Angular application
+matomo.tracker.url = http://localhost:8081
+# Specifies bitstream's bundle that will be tracked ( default is ORIGINAL )
+# Add 'none' to disable the tracking for bitstreams
+# matomo.track.bundles = ORIGINAL
+# Specifies custom cookies that will be tracked ( default is _pk_ref,_pk_hsr,_pk_ses ) directly from the request
+matomo.request.customcookies.cookie-name = _pk_ref,_pk_hsr,_pk_ses
```

Comment:
Because this configuration doesn't appear to be used in `dspace/config/spring/rest/matomo.xml` (it's commented out), I'd recommend commenting it out here as well. 

I also admit, I don't understand this setting, and what cookies these default settings are referencing.  So, it may need more documentation at some point.

Reply:
This settings are related to the cookies that can be generated on the Angular side.
However they're not usable since the cookies are not sent in CROSS-DOMAIN environments, and are disabled by default.
I can comment them out and add more details ( like I've done inside the matomo.xml file)